shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -457,39 +508,42 @@ async fn test_cassandra_peers_rewrite_cassandra3() {
     );
 
     let connection = shotover_manager
-        .cassandra_connection("127.0.0.1", 9044)
+        .cassandra_connection("127.0.0.1", 9044, driver)
         .await;
-    // run some basic tests to confirm it works as normal
-    table::test(&connection).await;
+    table::test(&connection).await; // run some basic tests to confirm it works as normal
 
     // Assert that the error cassandra gives because system.peers_v2 does not exist on cassandra v3
     // is passed through shotover unchanged.
     let statement = "SELECT data_center, native_port, rack FROM system.peers_v2;";
     let result = connection.execute_expect_err(statement);
-    assert!(matches!(
+    assert_eq!(
         result,
-        Error(
-            ErrorKind::CassErrorResult(cassandra_cpp::CassErrorCode::SERVER_INVALID_QUERY, ..),
-            _
-        )
-    ));
+        CassandraError {
+            code: 0x2200.into(),
```

Comment:
wouldnt it make sense to just use the enum variant name considering we went to the trouble of setting it up?